vllm/model_executor/weight_utils.py

```diff
@@ -286,3 +290,26 @@ def initialize_dummy_weights(
     for param in model.state_dict().values():
         if torch.is_floating_point(param):
             param.data.uniform_(low, high)
+
+
+def load_tensorized_weights(model: torch.nn.Module,
+                            model_config: ModelConfig):
+    before_mem = get_mem_usage()
+    # Lazy load the tensors from S3 into the model.
+    start = time.time()
```

Comment:
the timing stuff and the bragging aren't needed here, they're just illustrative in the tensorizer example.